packages/contracts/src/core/dao/DAO.sol

```diff
@@ -266,7 +266,7 @@ contract DAO is
 
     /// @notice Fallback to handle future versions of the [ERC-165](https://eips.ethereum.org/EIPS/eip-165) standard.
     /// @param _input An alias being equivalent to `msg.data`. This feature of the fallback function was introduced with the [solidity compiler version 0.7.6](https://github.com/ethereum/solidity/releases/tag/v0.7.6)
-    /// @return bytes The magic number registered for the function selector triggering the fallback.
+    /// @return The magic number registered for the function selector triggering the fallback.
```

Comment:
Isn't it expected to specify the type in curly braces so that it gets treated as the type? Not really sure, but worth checking

```suggestion
    /// @return {bytes} The magic number registered for the function selector triggering the fallback.
```

Reply:
The compiler returns the type automatically so we don't need to put it there ourselves. I checked it in the auto-generated docs so we can resolve this